docs/assets/css/custom.css

```diff
@@ -0,0 +1,23 @@
+.logo-config {
+  vertical-align: middle !important;
+  width: 100%;
+  height: auto;
+  width: calc(100px + 6 * ((100vw - 320px) / 680))
```

Comment:
Can you add a comment explaining this?

Reply:
I took the footer layout from by looking at this [example](https://bbbootstrap.com/snippets/simple-footer-newsletter-subscription-input-95837754). 

I just adjusted what fit with our logo. I [looked](https://alligator.io/css/viewport-units/#:~:text=vh%20%26%20vw,when%20the%20viewport%20is%20resized.) up w.r.t VW and adjusted the number. I will admit I totally don't know why it needs to be done this way.

Reply:
Hm, okay, I looked at this more. It seems like the end result is just that the image stays the same size. Couldn't we just do something like this and not use this CSS class?
```
        <h5><img src={{ "/assets/images/um-logo.png" | relative_url }} alt="UM logo" class="mb-1" width="105" height="114"></h5>
```